webapp/platform/client/src/client4.ts

```diff
@@ -3104,6 +3104,13 @@ export default class Client4 {
         );
     };
 
+    getAllPlainLogs = () => {
+        return this.doFetch<string[]>(
+            `${this.getBaseRoute()}/logs`,
+            {method: 'get'},
+        );
+    };
```

Comment:
I'm surprised that no pagination is needed, but this call seems to fetch the whole log file.